app/views/layouts/application.html.erb

```diff
@@ -30,6 +30,7 @@
     <% end %>
     <%= yield :javascript %>
     <%= javascript_include_tag "jquery.simulate" if Rails.env.test? %>
+    <!--[if lt IE 10]> <%= javascript_include_tag "flexibility" %> <![endif]-->
```

Comment:
I am downloading an IE10 VM. I will test it when it finishes, with and without -ms- prefix.